src/System.Private.Uri/tests/FunctionalTests/UriRelativeResolutionTest.cs

```diff
@@ -136,6 +136,16 @@ public void Uri_Relative_BaseVsSimplePartialPathWithScheme_ReturnsPartialPathWit
         }
 
         [Fact]
+        public void Uri_Relative_SimplePartialPathWithUnknownScheme_Unicode_ReturnsPartialPathWithScheme()
+        {
+            string schemeAndRelative = "scheme:\u011E";
+            Uri resolved = new Uri(schemeAndRelative, UriKind.RelativeOrAbsolute);
+
+            String expectedResult = schemeAndRelative;
+            Assert.Equal(expectedResult, resolved.ToString());
```

Comment:
And actually, there's no need for this variable at all. schemeAndRelative, which can also be made const and have it's first letter capitalized, can be passed as the first arg to Equal.